<launch>
	<arg name="com" default="/dev/ttyUSB0"/>
	<arg name="topicID" default="1"/>
	<arg name="baudrate" default="230400"/>
	<arg name="updateRate" default="20"/>
	<arg name="firstGen" default="" doc="description for this ary"/>
	<arg name="secondGen" default="" doc="description for this ary"/>
	<arg name="globalID" default="-1"/>
	<arg name="axisNum" default="1" />
	<node pkg="om_modbus_master" name="om_modbusRTU_$(arg topicID)" type="om_modbusRTU_node"  output="screen">
		<param name="init_com" type="str" value="$(arg com)"/>
		<param name="init_topicID" type="str" value="$(arg topicID)"/>
		<param name="init_baudrate" type="str" value="$(arg baudrate)"/>
		<param name="init_update_rate" type="str" value="$(arg updateRate)"/>
		<param name="first_gen" type="str" value="$(arg firstGen)"/>
		<param name="second_gen" type="str" value="$(arg secondGen)"/>
		<param name="global_id" type="str" value="$(arg globalID)"/>
		<param name="axis_num" type="str" value="$(arg axisNum)"/>
	</node>
	<node name="om_ros_node" pkg="om_modbus_master" type="om_ros_node" />
</launch>
